dspace/config/dspace.cfg

```diff
@@ -350,7 +350,7 @@ handle.dir = ${dspace.dir}/handle-server
 # Authorize community administrators to manage item settings (for items owned by collections under the community)
 #core.authorization.community-admin.item.delete = true
 #core.authorization.community-admin.item.withdraw = true
-#core.authorization.community-admin.item.reinstatiate = true
+#core.authorization.community-admin.item.reinstantiate = true
```

Comment:
This configuration key `core.authorization.community-admin.item.reinstantiate` is referenced in `AuthorizeConfiguration::canCommunityAdminPerformItemReinstatiate` - some DS instances could have been copied the key into `local.cfg`. If the key name is changed (corrected), a note needs to be added to the release notes that indicate this renaming.

Reply:
BTW: What is the correct term `reinstate` (more than 100 matches in the Java code base) or `reinstantiate` (currently, only 1 match)?

Reply:
BTW 2: Why is codespell complaining about `reinstatiate` in a string literal (in `AuthorizeConfiguration::canCommunityAdminPerformItemReinstatiate`) but not in a method name (`canCommunityAdminPerformItemReinstatiate`)? The same observation holds for `canCollectionAdminPerformItemReinstatiate`.

Reply:
The action in our documentation and other controls is "reinstate", I believe - it could be correct to call this 'an act of reinstantiation' but you raise a good point. I think we should change all instances of this to reinstate, and add a note about the configuration property change to the release notes / doco

Reply:
"Reinstate" seems correct, since it seems to be paired with "withdraw".  "Reinstantiate" ought to mean to create again from a type or template, and seems unlikely here.